moveit_setup_assistant/test/moveit_config_data_test.cpp

```diff
@@ -100,8 +100,12 @@ TEST_F(MoveItConfigData, ReadingControllers)
   // Test that addDefaultControllers() did accually add a controller for the new_group
   EXPECT_EQ(config_data_->getROSControllers().size(), group_count);
 
+  // Make a temp dir to write ros_controllers file to
+  char file_name[] = "/tmp/ros_controller.XXXXXX.yaml";
+  int fd = mkstemp(file_name);
```

Comment:
Locally it works without having to use `mkstemp`, and it writes the file to `/tmp`. Is there a way to test if it would work on the build server or not?

Reply:
Usually /tmp is writable for all users. 
There is no way to check whether it will work on Travis than testing on Travis ;-)
I was asking for deleting the file after you are done. (Always cleanup behind you.)

Reply:
Yeah thanks, I re-added the remove file command, I thought `mkstemp` will automatically delete it, but after testing, `mkstemp` didn't actually delete it.